cognite/neat/graph/transformers/_value_type.py

```diff
@@ -0,0 +1,110 @@
+from typing import cast
+
+from rdflib import XSD, Graph, URIRef
+
+from cognite.neat.issues._base import IssueList
+from cognite.neat.rules.analysis import InformationAnalysis
+from cognite.neat.rules.models._rdfpath import SingleProperty
+from cognite.neat.rules.models.data_types import AnyURI, DataType
+from cognite.neat.rules.models.entities import (
+    ClassEntity,
+    MultiValueTypeInfo,
+)
+from cognite.neat.rules.models.information import InformationRules
+
+from ._base import BaseTransformer
+
+
+class SplitMultiValueProperty(BaseTransformer):
```

Comment:
Unrelated to this PR: What about renaming BaseTransformer to GraphTransformer to distinguish from RuleTransformer?

Reply:
good point lets do that ! I will make maint. task in Jira board